src/Workspaces/Remote/ServiceHub/Services/Host/RemoteHostService.cs

```diff
@@ -60,10 +48,6 @@ static RemoteHostService()
                 Process.GetCurrentProcess().TrySetPriorityClass(ProcessPriorityClass.BelowNormal);
             }
 
-            // this is the very first service which will be called from client (VS)
-            // we set up logger here
-            RoslynLogger.SetLogger(new EtwLogger(s_logChecker));
```

Comment:
Not sure what's the point of setting a logger here with predicate that always returns `false` - meaning all Log calls are no-ops.